pkg/detectors/maxmindlicense/v2/maxmindlicense_v2.go

```diff
@@ -37,49 +37,41 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	dataStr := string(data)
 
 	keyMatches := keyPat.FindAllStringSubmatch(dataStr, -1)
-	idMatches := idPat.FindAllStringSubmatch(dataStr, -1)
 
 	for _, keyMatch := range keyMatches {
-		keyRes := strings.TrimSpace(keyMatch[1])
+		keyRes := keyMatch[1]
+
+		r := detectors.Result{
+			DetectorType: detectorspb.DetectorType_MaxMindLicense,
+			Redacted:     keyRes,
+			Raw:          []byte(keyRes),
+			ExtraData: map[string]string{
+				"rotation_guide": "https://howtorotate.com/docs/tutorials/maxmind/",
+				"version":        fmt.Sprintf("%d", s.Version()),
+			},
+		}
 
-		for _, idMatch := range idMatches {
-			if len(idMatch) != 2 {
+		if verify {
+			data := url.Values{}
+			data.Add("license_key", keyRes)
+			req, err := http.NewRequestWithContext(
+				ctx, "POST", "https://secret-scanning.maxmind.com/secrets/validate-license-key",
+				strings.NewReader(data.Encode()))
+			if err != nil {
 				continue
```

Comment:
It looks like other detectors use e.g `r.SetVerificationError(err, keyRes)` for the various errors

https://github.com/trufflesecurity/trufflehog/blob/1f0b53b530c50637d3a784e8a3fa1eb47097a464/pkg/detectors/accuweather/accuweather.go#L62-L67

Reply:
Although maybe it doesn't matter since we wouldn't be collecting anything with an error into `results` 🤔

Reply:
I think it should be fine AFAICT